datadog_checks_dev/datadog_checks/dev/config_validator/config_block.py

```diff
@@ -0,0 +1,443 @@
+# (C) Datadog, Inc. 2019
```

Comment:
Any tooling needs to go in https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev/datadog_checks/dev/tooling

Reply:
👍